operator/pkg/remote/kyma_synchronization_context.go

```diff
@@ -286,3 +288,15 @@ func GetRemoteObjectKey(kyma *v1alpha1.Kyma) client.ObjectKey {
 	}
 	return client.ObjectKey{Namespace: namespace, Name: name}
 }
+
+// InsertWatcherLabels inserts labels into the given KymaCR, which are needed to ensure
+// a working e2e-flow for the runtime-watcher.
+func (c *KymaSynchronizationContext) InsertWatcherLabels(remoteKyma *v1alpha1.Kyma) {
+	if remoteKyma.Labels == nil {
+		remoteKyma.Labels = make(map[string]string)
+	}
+
+	remoteKyma.Labels["operator.kyma-project.io/owned-by"] = fmt.Sprintf(v1alpha1.OwnerByFormat,
+		c.ControlPlaneKyma.Namespace, c.ControlPlaneKyma.Name)
+	remoteKyma.Labels["operator.kyma-project.io/watched-by"] = v1alpha1.OperatorName
```

Comment:
```suggestion
	remoteKyma.Labels[v1alpha1.OwnedBy] = fmt.Sprintf(v1alpha1.OwnerByFormat,
		c.ControlPlaneKyma.Namespace, c.ControlPlaneKyma.Name)
	remoteKyma.Labels[v1alpha1.WatchedBy] = v1alpha1.OperatorName
```

Reply:
Ah damn it, sure I also wanted to move them, but overlooked them 👍🏻